src/Helpers/helpers.php

```diff
@@ -63,4 +63,28 @@ function fireCmsEvent($eventName, $input = [])
         $method = method_exists(\Illuminate\Events\Dispatcher::class, 'dispatch') ? 'dispatch' : 'fire';
         Event::$method($eventName, [$eventName, $input]);
     }
-}
\ No newline at end of file
+}
+
+if (!function_exists('twill_path')) {
+    function twill_path($path = '')
+    {
+        // Split to separate root namespace
+        preg_match('/(\w*)\W?(.*)/', config('twill.namespace'), $matches);
+
+        // Namespace App is unchanged in config?
+        if ($matches[0] === 'App') {
+            return app_path($path);
+        }
+
+        // If it it still starts with App, use the left part, otherwise use the whole namespace
+        // This can be a problem for those using a completely different app path for the application
```

Comment:
Everything LGTM @antonioribeiro, I'd just like to discuss your comment here. Can you elaborate?

Reply:
Just an edge case for those not doing things the way most developers do these days, maybe. 

Let's say someone, insted of this 

	"psr-4": {
	    "App\\": "app/"
	},

Does this 

	"psr-4": {
	    "Area17\\App\\": "area17/app/"
	},

That namespace & path resolution may not work, since it takes in consideration everything starts with App or only the very first part of the namespace.

Reply:
Right, makes sense! I think that's perfectly fine. We might want to add a note in the config file about it, but like you said, very edge case.